cmd/amp/account.go

```diff
@@ -40,198 +55,759 @@ var (
 	switchRoleCmd = &cobra.Command{
 		Use:   "switch [ORGANIZATION]",
```

Comment:
This shouldn't be an optional parameter. And switch is for ACCOUNT, not ORGANIZATION. You might switch to the account that is the primary account directly associated with your account id (ie, your individual account), or you might switch to an organization account.